### ISSUE TITLE
Cargo-Audit Issues

### ISSUE THREAD

**Issue by dragazo** [NetsBlox/cloud#118]
The following two issues are reported from `cargo audit`:

```
Crate:     rusoto_credential
Version:   0.47.0
Warning:   unmaintained
Title:     Rusoto is unmaintained
Date:      2022-04-24
ID:        RUSTSEC-2022-0071
URL:       https://rustsec.org/advisories/RUSTSEC-2022-0071
Dependency tree:
rusoto_credential 0.47.0
├── rusoto_signature 0.47.0
│   ├── rusoto_core 0.47.0
│   │   ├── rusoto_s3 0.47.0
│   │   │   ├── netsblox-migrate 0.6.0
│   │   │   └── netsblox-cloud 0.6.0
│   │   ├── netsblox-migrate 0.6.0
│   │   └── netsblox-cloud 0.6.0
│   └── netsblox-cloud 0.6.0
└── rusoto_core 0.47.0

Crate:     atty
Version:   0.2.14
Warning:   unsound
Title:     Potential unaligned read
Date:      2021-07-04
ID:        RUSTSEC-2021-0145
URL:       https://rustsec.org/advisories/RUSTSEC-2021-0145
Dependency tree:
atty 0.2.14
├── env_logger 0.9.3
│   └── netsblox-cloud 0.6.0
└── clap 3.2.25
    ├── netsblox-migrate 0.6.0
    └── netsblox-cli 0.6.0
```

For `rusoto_credential`, they have some migration notes in their crate readme, not sure for `atty` (depending on what's being used, it could be replaced with a different alternative).

**Comment by dragazo** [comment]
I didn't notice `atty` was a transitive dependency (apparently their dep tree in the audit output is inverted). It's needed by `clap`, which we need, but apparenty upgrading clap would fix the problem. In the VM, I use `clap v4.4.3` and apparently they replaced `atty` with `std:io::IsTerminal`, thus fixing this audit issue. `env_logger` looks similarly fixable by upgrading the dep version, as they replaced `atty` with `is-terminal` in `v0.10.0`. Also apparently the `atty` dep is optional (enabled by default), so that could be an easier solution if upgrading the version is a breaking change.

**Comment by brollb** [comment]
Fixed by #119 . Thanks, @dragazo!
